eclass/cmake.eclass

```diff
@@ -698,6 +716,25 @@ cmake_src_install() {
 		einstalldocs
 		popd > /dev/null || die
 	fi
+
+	local file files=()
+	while read -d '' -r file ; do
+		# Detect unsupported minimum CMake versions unless CMAKE_QA_COMPAT_SKIP is set
+		if ! [[ ${CMAKE_QA_COMPAT_SKIP} ]]; then
+			_cmake_minreqver-detect "3.5" "${file}" || files+=( "${file}" )
+		fi
+	done < <(find "${D}" -type f -iname "*.cmake" -print0)
+	if [[ ${#files[*]} -gt 0 ]]; then
+		eqawarn "QA Notice: Package installs CMake module(s) incompatible with CMake 4,"
+		eqawarn "breaking any packages relying on it:"
+		eqawarn
+		for file in "${files[@]}"; do
+			eqawarn "    ${file/${D}/}"
```

Comment:
Wouldn't:

```suggestion
			eqawarn "    ${file#${D}}"
```

work?

Reply:
Also, quotes for whomever had the funny ideas to have globs in their dir names
```suggestion
			eqawarn "    ${file#"${D}"}"
```
As a minor optimization, just `${file}` here and do `files+=( "${file#"${D}"}" )` earlier on not to store ${D} for nothing.